### PR TITLE
docs: add Mechanical API link to Mechanical Scripting page

### LINKED ISSUE #970: Link scriptings.mechanical.docs.pyansys.com with the Mechanical docs
@klmcadams @dipinknair @koubaa - we need to add a section at the beginning of the current "Mechanical scripting" section that redirects you to scripting.mechanical.docs.pyansys.com..

### PR DESCRIPTION
**Pull request by klmcadams**
Closes #970

### CODE REVIEW

File: doc/source/user_guide_scripting/index.rst [hunk 1]
```diff
@@ -23,8 +23,9 @@ You could already perform scripting of Mechanical with Python from inside
 Mechanical. PyMechanical leverages the same APIs but allows you to run your
 automation from outside Mechanical.
 
-For comprehensive information on these APIs, see the `Scripting in Mechanical Guide`_ in the
-Ansys Help.
+For comprehensive information on these APIs, see the
+`Mechanical API Documentation <https://scripting.mechanical.docs.pyansys.com/version/dev/api/index.html>`_
+and the `Scripting in Mechanical Guide`_ in the Ansys Help portal.
```
Comment (RobPasMue) [inline doc/source/user_guide_scripting/index.rst:28]: This should be added with a ``.. note::`` so that users pay attention to it.

**Review by RobPasMue — CHANGES_REQUESTED** [review]

**Comment by RobPasMue** [comment]
@klmcadams - what's the status here?

**Review by RobPasMue — APPROVED** [review]

**Comment by klmcadams** [comment]
@RobPasMue does this look okay?

note

**Comment by RobPasMue** [comment]
Works for me - thanks @klmcadams!

**Pull request merged**